client/source/js/modules/admin/manage-users-ctrl.js

```diff
@@ -0,0 +1,21 @@
+define(['./module', 'angular', 'underscore'], function (module, angular, _) {
+  'use strict';
+  module.controller('AdminManageUsersController', function ($scope, $http, info, users, UserManager, modalService) {
+    $scope.users = _(users.data.users).filter(function (u) {
+      return u.email != UserManager.data.email;
+    });
+
+    $scope.deleteUser = function (user) {
+      modalService.confirm(function(){
+        user_id = user.id;
```

Comment:
I did not want to use reference to the user object in a closure  (i.e. function that is called after the delete returned and user object might in theory be gone), when I only need the id.